packages/web/src/pages/rewards-page/components/modals/ChallengeRewardsModal/ListenStreakChallengeModalContent.tsx

```diff
@@ -87,43 +85,38 @@ export const ListenStreakChallengeModalContent = ({
     </Flex>
   ) : null
 
-  return (
+  const readyToClaim =
+    userChallenge?.claimableAmount && userChallenge?.claimableAmount > 0 ? (
```

Comment:
same `!!` change here

Reply:
yeah can do, also for useMemo, not really needed for booleans since those dont trigger rerender and it's a simple comp, just heads up